src/main/resources/openapi.json

```diff
@@ -1,314 +1,1279 @@
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the subscribed consumer belongs.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the subscribed consumer for which you want to retrieve records.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "name": "timeout",
+                    "in": "query",
+                    "description": "The maximum amount of time in milliseconds that the HTTP Bridge spends retrieving records before timing out the request.",
+                    "required": false,
+                    "schema": {
+                        "type": "integer"
+                    }
+                },
+                {
+                    "name": "max_bytes",
+                    "in": "query",
+                    "description": "The maximum size, in bytes, of unencoded keys and values that can be included in the response. Otherwise, an error response with code 422 is returned.",
+                    "required": false,
+                    "schema": {
+                        "type": "integer"
+                    }
+                }
+            ]
         }
-      ]
     },
-    "/consumers/{groupid}/instances/{name}/positions/end": {
-      "post": {
-        "operationId": "seekToEnd",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
+    "components": {
+        "schemas": {
+            "Offset": {
+                "title": "Offset",
+                "required": [],
+                "type": "object",
+                "properties": {
+                    "partition": {
+                        "type": "integer"
+                    },
+                    "offset": {
+                        "type": "integer"
+                    },
+                    "topic": {
+                        "type": "string"
+                    }
+                }
+            },
+            "Consumer": {
+                "title": "Consumer",
+                "type": "object",
+                "properties": {
+                    "name": {
+                        "description": "The unique name for the consumer instance. The name is used in URLs. ",
+                        "type": "string"
+                    },
+                    "format": {
+                        "description": "The allowable message format for the consumer, which can be binary (default), avro or json. The messages are converted into a JSON format. ",
```

Comment:
We don't support avro. And imho you should use quotes ("binary") or monospace (if that's possible) if those are the names people should use in the request.

Reply:
...which can be `binary` (default) or `json`.